cfn/CI.yaml

```diff
@@ -53,6 +53,7 @@ Resources:
                     "arn:aws:iam::587316601012:role/service-role/codebuild-AWS-MPL-Java-service-role-release",
                     "arn:aws:iam::587316601012:role/service-role/codebuild-AWS-MPL-DotNet-service-role-release",
                     "arn:aws:iam::587316601012:role/service-role/codebuild-AWS-ESDK-DotNet-service-role-release",
+                    "arn:aws:iam::587316601012:role/service-role/codebuild-AWS-MPL-service-role-release",
```

Comment:
Blocking/Suggestion: This cannot be given to the CI Role or any PR CI can assume it.
We must create a new GitHub Role explicitly for releasing and assume that.
```suggestion
```

I know that we have other protections to prevent this abuse,
but least privileged strongly suggests that we keep this role/CFN template CI oriented.

Create a different IAM Role explicitly for releasing.
Do not allow `ToolsDevelopment` or any other CI role to assume it.

Reply:
I am confused. AFAIK this doesn't do that. This gives the codebuild service role permissions to assume the CI role used to test, It DOES NOT let the CI role assume the codebuild service role.

Reply:
Oh you are right!
I am an idiot.